content/en/istio-ca-certs-integrations/cert-manager-integration/_index.md

```diff
@@ -188,13 +184,170 @@ patches:
 
 The installation should complete, and the Istio control plane should become in a
 ready state.
-
 ```sh
-$ kubectl get pods -n istio-system
-NAME                                   READY   STATUS    RESTARTS   AGE
-istio-ingressgateway-9ddfd475f-47ssj   1/1     Running   0          45s
-istiod-6ff8d84d66-tvdl2                1/1     Running   0          71s
+kubectl get pods -n istio-system
 ```
 
+## Test mTLS
+
 All workload certificates will now be requested through cert-manager using the
-configured Issuer.
+configured Issuer. Let's run an example workload to test mTLS. First, create
+a namespace and configure it for automatic sidecar injection:
+```sh
+kubectl create ns foo
+kubectl label ns/foo istio-injection=enabled
+```
+
+Run the sample sleep and httpbin workloads.
+```sh
+ISTIO_VERSION=$(getmesh version | grep "control plane" | cut -d " " -f 4 | cut -d "." -f 1,2)
+kubectl apply -n foo -f https://raw.githubusercontent.com/istio/istio/release-$ISTIO_VERSION/samples/sleep/sleep.yaml
+kubectl apply -n foo -f https://raw.githubusercontent.com/istio/istio/release-$ISTIO_VERSION/samples/httpbin/httpbin.yaml
+```
+
+Verify the sleep and httpbin deployments have successfully rolled-out.
+```sh
+for i in sleep httpbin; do kubectl rollout status -n foo deploy/$i; done
+```
+
+Verify the sidecar proxy was injected for each workload. Each workload pod should
+show `2/2` containers are `READY`.
+```sh
+for i in sleep httpbin; do kubectl get po -n foo -l app=$i; done
+```
+
+By default, Istio configures destination workloads using `PERMISSIVE` mode. This
+mode means a service can accept both plain text and mTLS connections. To ensure
+mTLS is being used between sleep and httpbin workloads, set the mode to `STRICT`
+for namespace "foo".
+```sh
+kubectl apply -n foo -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: "default"
+spec:
+  mtls:
+    mode: STRICT
+EOF
+```
+
+Test mTLS from the sleep pod to the httpbin pod.
+```sh
+kubectl -n foo exec -it deploy/sleep -c sleep -- curl -o /dev/null -s -w '%{http_code}\n' http://httpbin.foo:8000/headers
+```
+You should receive an `200` response code.
+
+## Secure Gateways
```

Comment:
[nit] I think this section takes focus away from the main subject of this page.

Reply:
I removed this section.